app.js

```diff
@@ -64,13 +63,8 @@ app.use(passport.initialize())
 app.use(passport.session())
 app.use(flash())
 app.use(lusca.csrf())
-// app.use((req, res, next) => {
-//   if (req.path === '/api/upload') {
-//     next();
-//   } else {
-//     lusca.csrf(req, res, next);
-//   }
-// });
+app.use(lusca.xframe('SAMEORIGIN'))
+app.use(lusca.xssProtection(true))
```

Comment:
Why are these lines added again when they're already in the file?

Reply:
removed duplicates